api.yaml

```diff
@@ -209,6 +209,56 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
+  /v1/partitions:
```

Comment:
Question: Do you consider this part of the v1 API contract? Does that mean that PostgreSQL is not actually a choice and/or that a requirement for future implementations must support a concept of partition?

Reply:
I thought about not having it versioned, but did this mostly for consistency.

>  requirement for future implementations must support a concept of partition

It may make more sense to not version this

Reply:
Alternatively, we might have an `/ops` section where these things can go. I imagine that we _would_ have a versioned API contract for the endpoints but only have endpoints available based on the DB installed. For example, we might offer `/ops/pgsql/v1/...` for these cases.